Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        private const string CredentialsPath = @"..\..\..\Database\Credentials.txt";
+
+        public CredentialsManager()
+        {
+            if (!File.Exists(CredentialsPath))
+            {
+                File.WriteAllText(CredentialsPath, "");
+            }
+        }
+
+        public void Register()
```

Comment:
Register should accept user. Getting user input should come from outside- the console part. Don't couple console and business logic.

Reply:
Changed code for Register function to accept credentials.   The getting of the input was separated from the console part.  This was due to a misunderstanding that "Program.cs should just call what is already implemented" in a reply thinking that mean everything needs to be together inside.  Logic was separated to the way it was originally designed.